starter-core/src/main/java/io/micronaut/starter/feature/build/gradle/templates/buildGradle.rocker.raw

```diff
@@ -251,7 +251,7 @@ protobuf {
 protobuf {
     protoc { artifact = "com.google.protobuf:protoc:@VersionInfo.getBomVersion("protobuf")" }
     plugins {
-        grpc { artifact = "io.grpc:protoc-gen-grpc-java:@VersionInfo.getBomVersion("grpc")" }
+        grpc { artifact = "io.grpc:protoc-gen-grpc-java:@VersionInfo.getBomVersion("micronaut.grpc")" }
```

Comment:
Please, move this to a different PR.

Reply:
Done https://github.com/micronaut-projects/micronaut-starter/pull/1162